net/Kconfig

```diff
@@ -150,6 +150,11 @@ config NET_LOOPBACK_PKTSIZE
 		CONFIG_NET_LOOPBACK_PKTSIZE is zero, meaning that this maximum
 		packet size will be used by loopback driver.
 
+menuconfig NET_MBIM
+        bool "MBIM modem support"
+	depends on USBHOST_CDCMBIM
+	default n
+
```

Comment:
Something wrong here.  Looks like TABs used on some lines and spaces on others?

Reply:
Yes, it was with spaces in the first line, fixed!